Contents/title_part_common.tex

```diff
@@ -27,6 +27,37 @@
 	\scnaddlevel{-1}
 \scnaddlevel{-1}
 
+\scnheader{Стандарт OSTIS-2021}
+\scnidtf{Издание Документации Технологии OSTIS-2021}
```

Comment:
Maybe 2022?

Reply:
No, for 2022 there is correspondent part
This part is still about 2021 version